src/app/user/updatePost.js

```
@@ -0,0 +1,19 @@
+import httpStatus from 'http-status';
+import { updatePost } from '../../services/postsService';
+
+export default async function updateUserPost(req, res) {
+  try {
+    const {
+      postID, title, content, label,
+    } = req.body;
+    const response = await updatePost(postID, {
+      title, content, label,
+    });
+    if (response.includes(0)) {
```

Please do not include any logic in controller.

---

Hard to delete this logic, since updatePost always return success either the postID exist or not. If remove this logic, then it will always return 200 when no error catch, but we can include a field in the body, such as `updated: true/false`?

---

handle this situation in updatePost function?

---

updatePost function should return a value, then controller must check the return value to decide the response status. Or should updatePost function throw error if postID not found?

---

Please create a new service called `forumService`. I added this in my pr. We should never include and logic in controllers.

---

Logic move to postService